spec/controllers/catalog_controller_spec.rb

```diff
@@ -206,12 +200,8 @@ def year_created_facet
     end
 
     describe "year_created facet search" do
-
-      before do
```

Comment:
We use before hooks in alot of cases where it is only being used for one example. Why? Before blocks are meant for drying up duplication across multiple examples.